modules/machineset-yaml-vsphere.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-vsphere.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="machineset-yaml-vsphere_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on vSphere
+
+This sample YAML defines a MachineSet that runs on VMware vSphere and creates nodes that are labeled with `node-role.kubernetes.io/<role>: ""`.
+
+In this sample, `<infrastructureID>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and `<role>` is the node label to add.
```

Comment:
Following our replaceable value conventions, `<infrastructureID>` should really be `<infrastructure_id>`

Reply:
Yeah, good call. I was following the other MachineSet docs for consistency, but it's probably better to do it the right way here. I can follow-up with updating the others separately.